### ISSUE TITLE
shortcode before the summary tag not recognized when render the full page

### ISSUE THREAD

**Issue by Lynxiayel** [gohugoio/hugo#5464]
hugo version: 0.51
description: when using some shortcode before the  `<!--more-->` tag, the shortcode is not recognized when render the full page. 

I understand that when generating the summary, the shortcode may get ignored for a more simplistic and well organized "text-only" summary, but it should be recognized and interpreted when render the full page.

**Comment by bep** [comment]
As I doubt the statement in the title is true in general, I think you need to provide a failing test case (there are plenty of passing test cases for this particular issue in Hugo's code base).

**Comment by bep** [comment]
Yes, this is a bug, but you can work around it by inserting "something" (blank space, whatever) before your first shortcode.
